src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -104,11 +104,17 @@ const DocumentDetailsLayout = ({
             </ILSParagraphPlaceholder>
           </Container>
         </Container>
-        <Container textAlign="center">
-          {!_isEmpty(documentDetails.metadata) && (
-            <DocumentStats document={documentDetails} />
+        <AuthenticationGuard
```

Comment:
Are you sure about this? I think that if you are anonymous, it will try to redirect?

Reply:
that is the default behaviour but i skip that by using the loginComponent prop
```
loginComponent={() => null}
```

Reply:
yes indeed but the code is doing: 
```
if (isAnonymous) {
      return LoginComponent ? (
        <LoginComponent />
      ) : (
        <Redirect
```
so it should go in the `Redirect` branch?

Reply:
well 
```
() => null; 
```
Evaluates to false so it would just call an empty object <LoginComponent /> which basically does 'nothing', which is intended compared to redirecting somewhere.